images/shared-images-controller/Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/kubevirtci/bootstrap:v20221105-0d76701 as builder
+FROM quay.io/kubevirtci/bootstrap:v20221222-8f66e7e
```

Comment:
The `as builder` is required here - I think the script is dropping it?

Reply:
Thanks for the hint :slightly_smiling_face: